image/system_a/data/lang/Polski.json

```diff
@@ -745,5 +744,7 @@
   "app_meditation_countdown_desc": "Startuje za",
   "app_meditation_summary_total": "<text>Suma:<br /><token>$VALUE</token></text>",
   "app_meditation_summary_average": "Średnia/dzień:",
-  "app_meditation_summary_title": "<text>Ostatnie <token>$VALUE</token> dni</text>"
+  "app_meditation_summary_title": "<text>Ostatnie <token>$VALUE</token> dni</text>",
+  "app_bell_welcome_message": "<text>Mudita Harmony<br/>jest wyłączony</text>",
+  "app_bell_welcome_charge_message": "<text>Naładuj Harmony<br/>i kliknij lekko</text>"
```

Comment:
shouldn't it be `naciśnij` instead od `kliknij`?

Reply:
I have no preference. Note that these translations were added 2 years ago:
https://github.com/mudita/MuditaOS/commit/14b6fa84f77dd0fe3bcde81da268036478424d0d#diff-5e8cfa4b879d64bf25edc4bee9df74f3adddd5bf8796349d540d190f74e0e08f
and are consistent with english version `light click`  as opposed to `light press` or `press lightly`.

Reply:
So I'm going to leave it as it is since changing the old texts is not a part of this PR. Besides I think the differentiation between "deep press" and "light click" is here by design.